BunqSdk.Tests/Model/Generated/Object/NotificationUrlTest.cs

```diff
@@ -0,0 +1,221 @@
+        }   
+        
+        [Fact]
+        public void TestChatMessageAnnouncementModel()
+        {
+            ExecuteNotificationUrlTest(
+                JSON_PATH_CHAT_MESSAGE_ANNOUNCEMENT_MODEL,
+                typeof(ChatMessageAnnouncement),
+                GET_CHAT_MESSAGE_ANNOUNCEMENT
+                );
+        }        
+        
+        [Fact]
+        public void TestDraftPaymentModel()
+        {
+            ExecuteNotificationUrlTest(
+                JSON_PATH_DRAFT_PAYMENT_MODEL,
+                typeof(DraftPayment),
+                GET_DRAFT_PAYMENT
+                );
+        }      
+        
+        [Fact]
+        public void TestMasterCardActionModel()
+        {
+            ExecuteNotificationUrlTest(
+                JSON_PATH_MASTER_CARD_ACTION_MODEL,
+                typeof(MasterCardAction),
+                GET_MASTER_CARD_ACTION
+                );
+        }  
+        
+        [Fact]
+        public void TestMonetaryAccountBankModel()
+        {
+            ExecuteNotificationUrlTest(
+                JSON_PATH_MONETARY_ACCOUNT_BANK_MODEL,
+                typeof(MonetaryAccountBank),
+                GET_MONETARY_ACCOUNT_BANK
+                );
+        }    
+        
+        [Fact]
+        public void TestPaymentBatchModel()
+        {
+            ExecuteNotificationUrlTest(
+                JSON_PATH_PAYMENT_BATCH_MODEL,
+                typeof(PaymentBatch),
+                GET_PAYMENT_BATCH
+                );
+        }     
+        
+        [Fact]
+        public void TestRequestInquiryModel()
+        {
+            ExecuteNotificationUrlTest(
+                JSON_PATH_REQUEST_INQUIRY_MODEL,
+                typeof(RequestInquiry),
+                GET_REQUEST_INQUIRY
+                );
+        }     
+        
+        [Fact]
+        public void TestRequestResponseModel()
+        {
+            ExecuteNotificationUrlTest(
+                JSON_PATH_REQUEST_RESPONSE_MODEL,
+                typeof(RequestResponse),
+                GET_REQUEST_RESPONSE
+                );
+        }  
+        
+        [Fact]
+        public void TestScheduledInstanceModel()
+        {
+            ExecuteNotificationUrlTest(
+                JSON_PATH_SCHEDULE_INSTANCE_MODEL,
+                typeof(ScheduleInstance),
+                GET_SCHEDULE_INSTANCE
+                );
+        }     
+        
+        [Fact]
+        public void TestScheduledPaymentModel()
+        {
+            ExecuteNotificationUrlTest(
+                JSON_PATH_SCHEDULE_PAYMENT_MODEL,
+                typeof(SchedulePayment),
+                GET_SCHEDULE_PAYMENT
+                );
+        }       
+        
+        [Fact]
+        public void TestShareInviteBankModel()
+        {
+            ExecuteNotificationUrlTest(
+                JSON_PATH_SHARE_INVITE_BANK_INQUIRY_MODEL,
+                typeof(ShareInviteBankInquiry),
+                GET_SHARE_INVITE_BANK_INQUIRY
+                );
```

Comment:
So wazzup with the indentation of the hanging braces?

Reply:
They feel left out and just want to join the party 😎